tests/function-controller/testsuite/runtimes/nodejs.go

```diff
@@ -15,7 +15,17 @@ import (
 func BasicNodeJSFunction(msg string, rtm serverlessv1alpha1.Runtime) *function.FunctionData {
```

Comment:
Is that function used anywhere in the code after you've added `BasicNodeJSFunctionWithCustomDependency`?

Reply:
Yes, 
in light_runtimes.go line 110
in runtime.go line 82